docs/src/components/ShowForTs.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+import { useTabGroupChoice } from '@docusaurus/theme-common'
+import MDXContent from '@theme/MDXContent'
+
+interface Props {
+  children: React.ReactNode
+}
+
+/**
+ * Only renders this block if user has selected TS in the codeblocks
+ * @Note leave a blank space after opening the tag e.g.
+ *
+ * @example
+ * <ShowForTs>
+ * // {blank space}
+ * ### Mdx Formatted content
+ * </ShowForTs>
+ * **/
+
+export default function ShowForTs({ children }: Props) {
```

Comment:
How about `<ShowFor lang="ts">`? And then also adding support for `<ShowFor lang="js">`. Or is that premature? Better to wait and do that when we actually have a use-case for JS as well?

Another way to look at it I guess is, if/when we need the same for JS, do we want to make another component  `<ShowForJs>`?, or do we want to have just one and configure it with a prop?

Reply:
Thought about that - but I don't the think the reverse case is likely, since JS stuff usually applies to both. Can do the change, but I feel like having `<ShowForTs>` is better than `<ShowFor lang="ts">`

Reply:
Great. Just leave it like it is then :)